packages/frontend-api/src/Resources/config/graphql-types/LoginDecorator.types.yml

```diff
@@ -0,0 +1,11 @@
+LoginDecorator:
```

Comment:
_nitpick_ 🤓 

What about naming input objects like `LoginInputDecorator` and `LoginInput`? I think it helps knowing purpose of each type (when creating and consuming API)

Reply:
Hmm, I tried it and I don!t think so. Why? Whith naming as LoginInput query would look like `mutation {LoginInput(input: {email: \"no-reply@shopsys.com\", password: \"user123\"}) {accessToken}}` - this is too many input in my opinion :smile: